background/get-addon-settings.js

```diff
@@ -161,6 +161,14 @@ chrome.storage.sync.get([...ADDON_SETTINGS_KEYS, "addonsEnabled"], (storageItems
       }
     }
 
+    if (addonsEnabled["editor-dark-mode"] && addonSettings["editor-dark-mode"]["dots"] === false) {
+      // Transition v1.38 to v1.39
+      madeAnyChanges = true;
+      delete addonSettings["editor-dark-mode"]["dots"];
```

Comment:
Very minor but deleting the old setting should probably be last in case something errors out. Also settings without dashes can just use dot notation.